elasticsearch.md

```diff
@@ -920,6 +920,76 @@ After adding the `filter` rule \(using the block duplication strategy\).
       kibana_index: ".kibana_@{user}"
 ```
 
+#### `response_fields`
+
+This rule allows filtering Elasticsearch responses using defined list of fields. It works in very similar way to `fields` rule. However in contrast to `fields` rule, which filters out document fields, this rule filters out response fields. It **doesn't make use of Field Level Security \(FLS\)** and could be applied to every response returned by Elasticsearch.
+
+It could be configured in two modes:
+  * *whitelist* passing only defined fields
+  * *blacklist* filtering out defined fields.
```

Comment:
filtering out (removing) only the defined fields from the response JSON object

Reply:
I removed "JSON" because this rule can also filter yaml or binary cbor responses